tests/pytests/unit/modules/test_baredoc.py

```diff
@@ -0,0 +1,102 @@
+import pytest
+
+import salt.modules.baredoc as baredoc
+from tests.support.runtests import RUNTIME_VARS
+
+
+@pytest.fixture
+def configure_loader_modules():
+    return {
+        baredoc: {
+            "__opts__": {"extension_modules": RUNTIME_VARS.SALT_CODE_DIR},
```

Comment:
I believe we want to avoid RUNTIME_VARS now.

@s0undt3ch what do we use in place? maybe just `tmp_path`?

Reply:
That is correct @Ch3LL.

```
from tests.support import CODE_DIR
```
@frogunder Use this `CODE_DIR` variable.
Also, do we really need to set `extension_modules`? Does it need to be something specific? If not, use `tmp_path` as @Ch3LL suggested.

Reply:
Removed RUNTIME_VARS, hopefully this works better.